Find a functionally equivalent code:javax.obex.ClientOperation.continueOperation:COMMENT
Method Modifier: public      
Comment:/**
 * Continues the operation since there is no data to read.
 * @param sendEmpty <code>true</code> if the operation should send an empty
 * packet or not send anything if there is no data to send
 * @param inStream <code>true</code> if the stream is input stream or is
 * output stream
 * @throws IOException if an IO error occurs
 */

@@ -14,20 +14,27 @@
             if (mPrivateInput == null) {
                 mPrivateInput = new PrivateInputStream(this);
             }
-            sendRequest(0x03);
+            if (!mGetFinalFlag) {
+                sendRequest(0x03);
+            } else {
+                sendRequest(0x83);
+                if (mReplyHeader.responseCode != ResponseCodes.OBEX_HTTP_CONTINUE) {
+                    mOperationDone = true;
+                }
+            }
             return true;
         } else if (mOperationDone) {
             return false;

